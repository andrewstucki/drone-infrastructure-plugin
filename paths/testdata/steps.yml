---
kind: pipeline

steps:
  - name: include
    when:
      paths:
        include:
          - README.md
  - name: exclude
    when:
      paths:
        exclude:
          - README.md
      event:
        - push
        - tag
  - name: exclude-include
    when:
      paths:
        include:
          - README.md
        exclude:
          - README.md
      event:
        - push
        - tag
  - name: exclude-no-match
    when:
      paths:
        exclude:
          - no-match.md
  - name: include-no-match
    when:
      paths:
        include:
          - no-match.md
---
kind: secret
type: external
data:
  password: 'drone/docker#password'
  username: 'drone/docker#username'
