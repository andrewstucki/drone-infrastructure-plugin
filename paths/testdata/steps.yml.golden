name: ""
kind: pipeline
steps:
  - when:
        paths:
            include:
              - README.md
    name: include
  - when:
        paths:
            exclude:
              - README.md
        event:
            exclude:
              - '*'
    name: exclude
  - when:
        paths:
            exclude:
              - README.md
            include:
              - README.md
        event:
            exclude:
              - '*'
    name: exclude-include
  - when:
        paths:
            exclude:
              - no-match.md
    name: exclude-no-match
  - when:
        paths:
            include:
              - no-match.md
        event:
            exclude:
              - '*'
    name: include-no-match
trigger:
    event:
        exclude:
          - '*'
---
name: ""
kind: secret
data:
    password: drone/docker#password
    username: drone/docker#username
type: external
